fedn/network/api/v1/session_routes.py

```diff
@@ -485,8 +485,8 @@ def patch_session(id: str):
         data = request.get_json()
         data["session_id"] = id
 
-        session.patch_with(data, throw_on_extra_keys=False)
-
+        merged_data = {**existing_session.to_dict(), **data}
+        session.patch_with(merged_data, throw_on_extra_keys=False)
```

Comment:
I believe a neither solution would be to patch the existing session 
```suggestion
        existing_session.patch_with(data, throw_on_extra_keys=False)
```